README.md

```diff
@@ -2,52 +2,72 @@
 
 
-
-# CORE-V Embedded Class, 2-Stage Core
-
+## CVE2 Family, 2-Stage Embedded Class Cores
 [CVE2](https://github.com/openhwgroup/cve2) is a low-complexity, low-power, 32-bit, in-order RISC-V core with a 2-stage pipeline that implements RV32[E|I][M]C instruction set architecture for achieving high-energy efficiency on control-oriented, computationally limited applications. Starting as a fork of the lowRISC Ibex core, the CVE2 will be pared back to essential components and verified at industrial-grade.
 
 
-# CORE-V CVA5 FPGA optimized core
-
+## CVA5 Family, 5-Stage, Application Class Cores for FPGAs
 [CVA5](https://github.com/openhwgroup/cva5) The CVA5 is a 32-bit RISC-V processor designed for FPGAs supporting the Multiply/Divide and Atomic extensions (RV32IMA). The processor is written in SystemVerilog and has been designed to be both highly extensible and highly configurable. The CVA5 is derived from the Taiga Project from Simon Fraser University. The pipeline has been designed to support parallel, variable-latency execution units and to readily support the inclusion of new execution units.
+
+
+# CORE-V Core Names and Releases
+Here is an explainer of how [CORE-V Device Names](https://docs.google.com/presentation/d/1MIe9b10-HRglDa67s8oc_2TTYux-WtMY/edit?usp=sharing&ouid=112401338207375688045&rtpof=true&sd=true) are determined.
+Any CORE-V core that achieves its target Technical Readiness Level (TRL) spawns a major release (also known as "RTL Freeze") of its host GitHub repository.
+A good discussion regarding the rules for creating and updating releases is given [here](https://docs.openhwgroup.org/projects/cv32e40p-user-manual/en/latest/core_versions.html#core-versions-and-rtl-freeze-rules).
+At the time of release (freeze), the values `mvendorid`, `marchid` and `mimpid` CSRs are set:
+- `mvendorid` is determined by [JEDEC](https://www.jedec.org/).  For OpenHW CORE-V devices the value is `0x602`.
+- `marchid` is determined by [RISC-V International](https://github.com/riscv/riscv-isa-manual/blob/main/marchid.md).
+A unique `marchid` is provided for each device.
+- `mimpid` is determined by OpenHW according to the above rules. The value will be specified in the User Manual.
+
+The Table below summarizes the "ID" values for the CORE-V cores that have been assigned an architecture ID from RVI:
+
+| CORE-V Name | Release | mvendorid  | marchid    | mimpid     | TRL Goal | TRL Achieved | Comment               |
+| ----------- | ------- | ---------- | ---------- | ---------- | -------- | ------------ | --------------------- |
+| CV32A60X    | TBD     | 0x00000602 | 0x00000003 | 0x00000000 | TRL-5    | Q1, 2025     |                       |
+| CV32E40Pv1  | v1.0.0  | 0x00000602 | 0x00000004 | 0x00000000 | TRL-5    | Yes          |                       |
```

Comment:
Suggestion to add a comment that some features/extensions (floating point, Xpulp) are not verified in v1, so not considered to be part of the core.

Reply:
Done!